### ISSUE TITLE
🔤 N/A used inconsistently and ambiguously

### ISSUE THREAD

**Issue by mnigh** [GCTC-NTGC/gc-digital-talent#8898]
## 🔤 Copy

The term **N/A** is used inconsistently on the site and is ambiguous since it can mean not applicable or not available.

### Screenshot

<img width="475" alt="Screen Shot 2023-12-22 at 09 47 44" src="https://github.com/GCTC-NTGC/gc-digital-talent/assets/3046459/e3655442-a02f-4903-9d14-fedf57feab71">

## 🙋‍♀️ Proposed Implementation

Replace all instances of **N/A** with **Not applicable** or **Not available** to avoid ambiguity of the abbreviation. Consolidate existing strings so that they use a single source of truth for each term. There are already strings for this but they need to be translated again.

https://github.com/GCTC-NTGC/gc-digital-talent/blob/f6b0ce8cab1a021cd178995d2ee5bb03c4403fc0/packages/i18n/src/messages/commonMessages.ts#L56-L61

https://github.com/GCTC-NTGC/gc-digital-talent/blob/f6b0ce8cab1a021cd178995d2ee5bb03c4403fc0/packages/i18n/src/messages/commonMessages.ts#L138-L142

## ✅ Acceptance Criteria

A set of assumptions which, when tested, verify that the copy has been added or updated appropriately.

- [ ] no instances of **N/A** appear in the codebase
- [ ] **Not applicable** translated in French correctly
- [ ] **Not available** translated in French correctly

**Comment by gobyrne** [comment]
I vote for just spelling out "Not applicable" everywhere

**Comment by gobyrne** [comment]
@substrae makes the point that is a well defined dictionary abbreviation and there are potentially some places where the longer text will affect the designs.

Sorry for the last minute change @mnigh , but can you please wrap this in a abbreviation tag instead? Then we can use "not applicable" or "not available" as appropriate in the tag.

**Comment by mnigh** [comment]
> the point that is a well defined dictionary abbreviation

Yes, n/a appears in the dictionary, but not applicable **and** not available are listed in the [cambridge dictionary](https://dictionary.cambridge.org/dictionary/english/n-a).

## Screenshot
<img width="755" alt="Screen Shot 2024-03-21 at 14 53 58" src="https://github.com/GCTC-NTGC/gc-digital-talent/assets/3046459/76c8a6d6-6221-4626-b711-b24d6afbec34">

> there are potentially some places where the longer text will affect the designs.

The only UI change that doesn't render well is on the [`PoolCard`](https://www.chromatic.com/component?appId=61e099a1bb1465003a73d3ce&csfId=components-pool-card&specName=Individual+Pool+Card&buildNumber=6771&k=65f8a25a0862471644b2464f-1200px-interactive-true&h=1&b=-1) where the text ribbon part doesn't fit a long string. Whether it is N/A or Not available, the text in the ribbon is duplicative since the same string appears right beside it so it offers nothing that isn't already there. Also, the `PoolCard` null state is a very edge case since it would only be rendered if there was no pool passed to the pool card (which would likely be an error in the underlining code).

## Screenshot

<img width="1041" alt="Screen Shot 2024-03-21 at 15 13 21" src="https://github.com/GCTC-NTGC/gc-digital-talent/assets/3046459/d48b305e-077e-4b23-b3f7-e1186b1de14d">

> can you please wrap this in a abbreviation tag instead

@gobyrne

With an accessibility lens, [WCAG](https://www.w3.org/TR/UNDERSTANDING-WCAG20/meaning-located.html#:~:text=Sometimes%2C%20the%20same%20abbreviation%20means%20different%20things%20in%20different%20contexts) rightly states that abbreviations can confuse some readers. If we can help it, why wouldn't we try to avoid confusing readers? From a UX perspective, adding an abbreviation tag delays clarity. The abbreviation tag forces users to hover to see the word, which is an added step instead of having the original word available from the start in concrete language.

**Comment by substrae** [comment]
@mnigh 

Couple things here:
* This concern should have been brought to the design team first, rather than this discussion happening mid-PR
* We're not suggesting you re-replace all the changes you've made, we're asking to have the banner version remain N/A and have it wrapped in an abbreviation tag
* Where applicable, it's fair for the design team to use common abbreviations to make the UI clearer when there is limited space, especially given that there are tools to make abbreviations easier to understand already in place
* We use abbreviations in plenty of locations on the site, deciding that this one specific location is suddenly one too many doesn't make much sense to me

**Comment by gobyrne** [comment]
@mnigh did bring it to me and I asked him to move it forward without going to design team. So my bad on that @substrae

On WCAG, I feel like we've had a tendency to extrapolate guidance outside of it's specific application. Indeed, the text you linked to @mnigh is actually describing the abbr tag as the solution, not the cause of the problem.

Anyway, @mnigh this is approved from our side, with the guidance that any place it affects the design of the page (by changing size of the copy) N/A should be left and abbr tag added.

Go forth and copy update!
